lib/WebpackConfig.js

```diff
@@ -49,6 +49,7 @@ class WebpackConfig {
         this.providedVariables = {};
         this.babelConfigurationCallback = function() {};
         this.useReact = false;
+        this.loaders = new Set();
```

Comment:
This does not need to be a Set. It could just be an array.

You create a new object inside `addLoader`, so the uniqueness check of the Set will never exclude anything anyway. You're just using it as an array here